rpm_requirements.txt

```diff
@@ -0,0 +1,10 @@
+bzip2
+git
+GitPython
+lzop
+mock
+nosync
+python-lxml
+PyYAML
+subversion
```

Comment:
The correct rpm package name is subversion. There is a separate commit in the PR which fixes that.